src/pudl/metadata/sources.py

```diff
@@ -835,5 +835,41 @@
         "license_raw": LICENSES["us-govt"],
         "license_pudl": LICENSES["cc-by-4.0"],
     },
+    "vceregen": {
+        "title": "Vibrant Clean Energy Renewable Generation Profiles",
+        "path": "https://vibrantcleanenergy.com/products/datasets/",
+        "description": (
+            "Hourly, county-level renewable generation profiles in the continental "
+            "United States. Compiled by Vibrant Clean Energy based on outputs from the "
+            "NOAA HRRR weather model. Profiles are stated as a capacity factor (a "
+            "fraction of nameplate capacity) and exist for onshore wind, offshore "
+            "wind, and fixed-tilt solar generation types."
+        ),
+        "source_file_dict": {
+            "sorce_format": "Comma Separated Value (.csv)",
+        },
+        "keywords": sorted(
+            {
+                "solar",
+                "wind",
+                "time series",
+                "energy",
+                "electricity",
+                "generation",
+                "weather",
+                "capacity factor",
+                "hourly",
+                "united states",
+                "usa",
+                "resource adequacy",
+                "gridpath",
+                "vibrant clean energy",
+                "county",
+            }
+        ),
+        "license_raw": LICENSES["cc-by-4.0"],
```

Comment:
I assumed it would be the same as `gridpathtoolkit`

Reply:
Yes, we got them to agree to CC-BY-4.0 after a bit of discussion. We need to make sure that the dataset is clearly attributed to VCE in the Zenodo archive. And also in the metadata that accompanies the PUDL outputs. This isn't something we've had to deal with in the past because all the input data has come from the US Govt which doesn't have copyrights.

Reply:
Ah, that's not in the licensing, that's in the contributors - we need to define a new organization in the CONTRIBUTOR field for VCE and define it here.

Reply:
Yeah, that sounds right!  I think so long as VCE has the "creator" role or whatever is equivalent, there's a CC-BY-4.0 license, they're mentioned explicitly in the description (which ends up in the description on Zenodo) and linked to as well, and in the Data Sources page in our docs, and we get analogous appropriate metadata into our Data Release archives on Zenodo, we'll be okay.